server/src/application/application.service.ts

```diff
@@ -71,4 +74,19 @@ export class ApplicationService {
   ): Promise<void> {
     await this.applicationRepository.update(applicationId, applicationDto);
   }
+
+  async assignToUser(applicationId: string, assignToUserDto: AssignToUserDto): Promise<void> {
+    const application = await this.getApplication(applicationId);
+    if (application) {
+      const user = await this.userService.getByExternalId(assignToUserDto.externalId);
+      if (user) {
+        application.user = user;
+        application.assignedTo = user.externalId;
+      } else {
+        application.user = null;
```

Comment:
We'll have a separate endpoint for unassign!

Reply:
Added /unassign endpoint